ocs_ci/helpers/dr_helpers_ui.py

```
@@ -915,13 +959,149 @@ def check_or_assign_drpolicy_for_discovered_vms_via_ui(
+        log.warning(
+            "Protection removal confirmation message not found on UI. "
+            "Will rely on backend DRPC verification."
+        )
+        acm_obj.take_screenshot()
+
+    log.info("Close the dialog")
+    acm_obj.do_click(acm_loc["close-page"], enable_screenshot=True)
+
+    log.info(
+        f"DR protection successfully removed from VM '{vm.vm_name}' "
+        f"in namespace '{namespace}'"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)